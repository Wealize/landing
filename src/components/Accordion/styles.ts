import styled from 'styled-components'

import { sixthPx, halfPx, twoThirdPx, singlePx, singleAndHalfPx, doublePx, doubleAndHalfPx, triplePx, quadruplePx, quadruple } from '../../theme/space'
import { media } from '../../theme/media'

export const Container = styled.div<{ backgroundColor: string, textColor: string}>`
  width: 100%;
  min-height: 20vh;
  padding: ${doubleAndHalfPx()} ${singlePx()};

  ${media.greaterThan('md')`
    padding: ${triplePx()} ${quadruplePx()};
  `};

  ${media.greaterThan('lg')`
    padding: ${quadruplePx()} ${quadruple() * 1.5}px;
  `};

  ${media.greaterThan('xl')`
    padding: ${quadruplePx()} ${quadruple() * 2.5}px;
  `};

  ${media.greaterThan('xxl')`
    padding: ${quadruplePx()} ${quadruple() * 3.5}px;
  `};

  color: ${({ textColor }) => textColor};
  background-color: ${({ backgroundColor }) => backgroundColor};
`

export const TitleContainer = styled.div`
  padding-bottom: 0;

  > .accordion-separator{
    margin-top: ${halfPx()};
  }
`

export const Title = styled.h5<{ textColor: string}>`
  font-size: 1rem;
  text-transform: uppercase;
  font-weight: bold;
  color: ${({ textColor }) => textColor};

  ${media.greaterThan('xl')`
    font-size: 1.1rem;
  `};

  ${media.greaterThan('lg')`
    font-size: 1.2rem;
  `};

  ${media.greaterThan('xxl')`
    font-size: 1.2rem;
  `};
`

export const Separator = styled.hr<{ textColor: string}>`
  width: 100%;
  background-color: ${({ textColor }) => textColor};
  opacity: .6;
  border: none;
  height: 2px;
  margin: 0;
`

export const SectionContainer = styled.div`
  padding-top: ${singleAndHalfPx()};

  > .accordion-separator {
    margin-top: ${singleAndHalfPx()};
  }

  ${media.greaterThan('md')`
    padding-top: ${doublePx()};

    > .accordion-separator {
      margin-top: ${doublePx()};
    }
  `};
`

export const SectionHeader = styled.div<{isExpanded: boolean}>`
  display: flex;
  width: 100%;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  padding-bottom: 0;

  > .button-rounded {
    display: none;

    & span {
      font-weight: lighter;
    }
  }


  ${media.greaterThan('lg')`
    > .accordion-section-toggle-button {
      display: none;
    }

    > .button-rounded {
      display: block;
      min-height: 75px;
    }
  `};

`

export const SectionTitle = styled.h6`
  font-size: 1.7rem;
  width: 75%;
  line-height: 1.52;

  ${media.greaterThan('md')`
    font-size: 1.8rem;
  `};

  ${media.greaterThan('lg')`
    font-size: 2.2em;
  `};

  ${media.greaterThan('xl')`
    font-size: 2.4rem;
  `};

  ${media.greaterThan('xxl')`
    font-size: 2.7rem;
  `};
`

export const SectionToggle = styled.button<{
  textColor: string
  backgroundColor: string
  isExpanded: boolean
}>`
  width: 40px;
  height: 40px;
  background-color: transparent;
  border: none;
  padding: 0;
  transition: all 0.5s ease-out;
  transform: ${({ isExpanded }) =>
    isExpanded ? 'rotate(0deg)' : 'rotate(180deg)'};

  & svg > g:first-child {
    fill: ${({ textColor, isExpanded }) => isExpanded ? textColor : 'none'};
  }

  & svg > g > path:last-child {
    fill: ${({ textColor }) => textColor};
  }

  & svg > path {
    fill: ${({ textColor, backgroundColor, isExpanded }) => isExpanded ? backgroundColor : textColor};
  }

  ${media.greaterThan('md')`
      width: 50px;
      height: 50px;
  `};

  ${media.greaterThan('lg')`
    display: none;
  `};
`

export const SectionContentContainer = styled.div<{isExpanded: boolean}>`
  width: 100%;
  display: grid;
  grid-template-columns: repeat(2, 1fr);
  grid-column-gap: ${singlePx()};
  transition: all 0.5s ease-out;
  overflow: hidden;
  height: auto;
  max-height: ${({ isExpanded }) => (isExpanded ? 1000 : 0)}px;
  padding-top: ${({ isExpanded }) => (isExpanded ? singleAndHalfPx() : 0)};

  ${media.greaterThan('lg')`
    grid-template-columns: repeat(3, 1fr);
  `};

  ${media.greaterThan('xl')`
    grid-template-columns: repeat(3, 1fr);
  `};

  ${media.greaterThan('xxl')`
    grid-template-columns: repeat(4, 1fr);
  `};

`

export const SectionElement = styled.span`
  padding: ${twoThirdPx()} 0 ${sixthPx()};
  line-height: 1.4;

  &:before {
    content: '- '
  }

  ${media.greaterThan('md')`
    font-size: 1.2rem;
    padding: ${twoThirdPx()};
  `};

  ${media.greaterThan('lg')`
    font-size: 1.3rem;
  `};

  ${media.greaterThan('xl')`
    font-size: 1.4rem;
  `};

  ${media.greaterThan('xxl')`
    font-size: 1.5rem;
  `};
`
