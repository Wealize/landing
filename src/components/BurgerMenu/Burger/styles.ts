import styled from 'styled-components'

import { ACCENT_COLOR } from '../../../theme/color'
import { media } from '../../../theme/media'
import { singlePx } from '../../../theme/space'

export const StyledBurger = styled.button<{ open: boolean }>`
  top: ${singlePx()};
  margin-top: 2.5%;
  left: ${({ open }) => (open ? '85%' : '85%')};
  width: 2rem;
  height: 1.5rem;
  position: fixed;
  background: transparent;
  z-index: 10;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
  border: none;
  cursor: pointer;
  padding: 0;
  &:focus {
    outline: none;
  }

  ${media.greaterThan('md')`
    margin-top: 0.4%;
  `}

  ${media.greaterThan('lg')`
    display: none;
  `}

  div {
    width: 1.5rem;
    height: 0.18rem;
    border-radius: 10px;
    transition: all 0.3s linear;
    position: relative;
    transform-origin: 1px;
    background-color: ${ACCENT_COLOR};

    :first-child {
      transform: ${({ open }) => (open ? 'rotate(45deg)' : 'rotate(0)')};
    }
    :nth-child(2) {
      opacity: ${({ open }) => (open ? '0' : '1')};
      transform: ${({ open }) => (open ? 'translateX(20px)' : 'translateX(0)')};
    }
    :nth-child(3) {
      transform: ${({ open }) => (open ? 'rotate(-45deg)' : 'rotate(0)')};
    }
  }
`
