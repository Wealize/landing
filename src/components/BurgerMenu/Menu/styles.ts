import styled from 'styled-components'

import {
  ACCENT_COLOR,
  DARK_GRAY_COLOR,
  WHITE_COLOR
} from '../../../theme/color'
import { media } from '../../../theme/media'

export const StyledMenu = styled.nav<{ open: boolean }>`
  position: fixed;
  top: 0;
  left: 0;
  height: 100%;
  width: 100vw;
  padding-left: 2em;
  padding-top: 20vh;
  background-color: ${WHITE_COLOR};
  display: flex;
  flex-direction: column;
  z-index: 5;
  transition: transform 0.3s ease-in-out;
  transform: ${({ open }) => (open ? 'translateY(0%)' : 'translateY(-100%)')};

  ${({ open }) => media.greaterThan('md')`
    transform: ${open ? 'translateX(65%)' : 'translateX(100%)'};
    padding-top: 14vh;
  `}

  ${media.greaterThan('lg')`
    display: none;
  `}
`

export const StyledLink = styled.a<{ isCurrentPage: boolean }>`
  font-weight: bold;
  text-decoration: none;
  color: ${DARK_GRAY_COLOR};
  font-size: 1.125rem;
  cursor: pointer;
  margin-bottom: 32px;
  max-width: fit-content;
  border-bottom: 3px solid
    ${({ isCurrentPage }) => (!isCurrentPage ? 'transparent' : ACCENT_COLOR)};

  transition: border-bottom 0.6s;

  & :hover {
    border-bottom: 3px solid ${ACCENT_COLOR};
  }
`
