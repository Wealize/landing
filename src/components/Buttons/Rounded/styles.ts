import styled from 'styled-components'

import { twoThirdPx, singlePx } from '../../../theme/space'
import { media } from '../../../theme/media'
import { ButtonSize } from '../../../interfaces/Buttons/ButtonSize'

const BUTTON_SHARED_STYLES = `
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  width: 100%;
  height: 100%;
  border: none;
  position: absolute;
  bottom: 0;
  transition: all ease-out 0.3s;
  cursor: pointer;
`

export const ContainerButtonOriginal = styled.button<{
  backgroundColor: string
}>`
  ${BUTTON_SHARED_STYLES};

  top: 0;
  background-color: ${({ backgroundColor }) => backgroundColor};
`

export const ContainerButtonAnimated = styled.button<{
  animatedBackgroundColor: string
}>`
  ${BUTTON_SHARED_STYLES};

  background-color: ${({ animatedBackgroundColor }) => animatedBackgroundColor};
  top: 150%;
  left: 0;
`

export const Content = styled.span<{ textColor: string }>`
  width: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  padding: ${twoThirdPx()};
  background-color: transparent;
  color: ${({ textColor }) => textColor};
  font-size: 1rem;
  font-weight: bold;
  border-radius: 50px;

  ${media.greaterThan('md')`
    padding: ${singlePx()};
  `};

  ${media.greaterThan('lg')`
    font-size: 1.2rem;
  `};

  ${media.greaterThan('xl')`
    font-size: 1.4rem;
  `};
`

export const Container = styled.div<{
  backgroundColor: string
  textColor: string
  animatedBackgroundColor: string
  animatedTextColor: string
  size: ButtonSize
}>`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;

  border-radius: 50px;
  border: 2px solid ${({ textColor }) => textColor};

  min-width: ${({ size }) => size.mobile[0]}px;
  min-height: ${({ size }) => size.mobile[1]}px;
  transition: all 0.3s;
  position: relative;
  overflow-y: hidden;
  overflow-x: hidden;
  cursor: pointer;
  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);

  ${({ size }) => media.greaterThan('md')`
    min-width: ${size.tablet[0]}px;
  `};

  ${({ size, animatedTextColor }) => media.greaterThan('lg')`
    min-width: ${size.desktop[0]}px;
    min-height: ${size.desktop[1]}px;

    &:hover {
      border: none;
    }

    &:hover ${ContainerButtonOriginal} {
      top: -120%;
    }

    &:hover ${ContainerButtonAnimated} {
      top: 0%;
      border: none;
    }

    &:hover ${Content} {
      color: ${animatedTextColor};
    }
  `};

  ${({ size }) => media.greaterThan('xl')`
    min-width: ${size.wide[0]}px;
    min-height: ${size.wide[1]}px;
  `};
`
