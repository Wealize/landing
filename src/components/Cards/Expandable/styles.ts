import styled from 'styled-components'

import {
  singlePx,
  singleAndHalfPx,
  doublePx,
  triplePx,
  quadruplePx,
  quadruple
} from '../../../theme/space'
import { media } from '../../../theme/media'

export const Container = styled.div<{
  backgroundColor: string
  textColor: string
  isExpanded: boolean
}>`
  width: 100%;
  display: flex;
  flex-direction: column;
  padding: ${singleAndHalfPx()} ${singlePx()};
  background-color: ${({ backgroundColor }) => backgroundColor};
  color: ${({ textColor }) => textColor};
  transition: all 0.5s ease-out;

  ${media.greaterThan('md')`
    padding: ${singleAndHalfPx()} ${quadruple() * 2}px;
  `};

  ${media.greaterThan('lg')`
    padding: ${quadruplePx()} ${doublePx()};
    min-height: 900px;
    min-width: 110%;
    justify-content: space-between;
  `};

  ${media.greaterThan('xl')`
    min-height: 980px;
  `};

  ${media.greaterThan('xxl')`
    padding: ${quadruplePx()};
    min-height: 1100px;
  `};
`

export const Header = styled.div<{
  isExpanded: boolean
}>`
  width: 100%;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
  padding-bottom: ${({ isExpanded }) => (isExpanded ? singlePx() : 0)}px;
`

export const Content = styled.div<{
  isExpanded: boolean
}>`
  flex-direction: column;
  justify-content: center;
  transition: max-height 0.5s ease-out;
  overflow: hidden;
  height: auto;
  max-height: ${({ isExpanded }) => (isExpanded ? 1000 : 0)}px;

  ${media.greaterThan('lg')`
    max-height: 1000px;
  `};
`

export const Headline = styled.h3`
  font-size: 1.2rem;
  font-weight: bold;

  ${media.greaterThan('md')`
    font-size: 1.2rem;
  `};

  ${media.greaterThan('lg')`
    font-size: 1.3rem;
  `};

  ${media.greaterThan('xl')`
    font-size: 1.5rem;
  `};

  ${media.greaterThan('xxl')`
    font-size: 2rem;
  `};
`

export const Toggle = styled.button<{
  textColor: string
  isExpanded: boolean
}>`
  width: 40px;
  height: 40px;
  background-color: transparent;
  border: none;
  padding: 0;
  transition: all 0.5s ease-out;
  transform: ${({ isExpanded }) =>
    isExpanded ? 'rotate(0deg)' : 'rotate(180deg)'};

  & svg > g > path:last-child {
    fill: ${({ textColor }) => textColor};
  }

  & svg > path {
    fill: ${({ textColor }) => textColor};
  }

  ${media.greaterThan('md')`
      width: 50px;
      height: 50px;
  `};

  ${media.greaterThan('lg')`
    display: none;
  `};
`

export const ImageContainer = styled.div`
  width: 100%;
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  padding: ${doublePx()} 0;

  & svg {
    width: 40%;

    ${media.greaterThan('md')`
      width: 55%;
    `};

    ${media.greaterThan('lg')`
      width: 65%;
    `};
  }
  ${media.greaterThan('md')`
    padding: ${triplePx()} 0;
  `};

  ${media.greaterThan('lg')`
    padding: ${quadruplePx()} 0;
  `};
`

export const Subheadline = styled.h4`
  display: flex;
  line-height: 1.2;
  flex-direction: row;
  justify-content: flex-start;
  font-size: 1.8rem;
  text-align: left;
  padding-bottom: ${singlePx()};

  ${media.greaterThan('md')`
    font-size: 1.7rem;
  `};

  ${media.greaterThan('lg')`
    font-size: 1.8rem;
  `};

  ${media.greaterThan('xl')`
    font-size: 2.2rem;
  `};

  ${media.greaterThan('xxl')`
    font-size: 2.5rem;
  `};
`

export const Description = styled.h5`
  font-size: 1.1rem;
  text-align: left;
  line-height: 1.43;

  ${media.greaterThan('md')`
    font-size: 1.3rem;
  `};

  ${media.greaterThan('lg')`
    font-size: 1.3rem;
  `};

  ${media.greaterThan('xl')`
    font-size: 1.7rem;
  `};
`
