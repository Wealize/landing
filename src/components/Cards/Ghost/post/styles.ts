import styled from 'styled-components'

import { WHITE_COLOR } from '../../../../theme/color'
import { singlePx } from '../../../../theme/space'

export const Container = styled.a`
  width: 100%;
  background: ${WHITE_COLOR};
  z-index: 1;
`

export const CardHeadline = styled.h4`
  font-size: 1rem;
  font-weight: bold;
  padding-bottom: ${singlePx()};
`

export const CardPublishedAt = styled.span`
  font-size: 1rem;
  padding: ${singlePx()} 0;
`
