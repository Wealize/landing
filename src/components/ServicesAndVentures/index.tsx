import React from 'react'
import useTranslation from 'next-translate/useTranslation'

import servicesData from '../../data/servicesData'
import venturesData from '../../data/venturesData'
import { ServiceContent } from '../../interfaces/Data/ServiceContent'
import { VentureContent } from '../../interfaces/Data/VentureContent'

import {
  Container,
  Title,
  ContentContainer,
  SectionContainer,
  SectionTitle,
  CardContainer,
  CardHeadline,
  CardDescription,
  Subtitle
} from './styles'
const ServicesAndVentures = () => {
  const { t } = useTranslation('profile')

  return (
    <Container>
      <Title>{t('services-ventures.title')}</Title>
      <Subtitle>{t('services-ventures.subtitle')}</Subtitle>
      <ContentContainer>
        <SectionContainer>
          <SectionTitle>{t('services-ventures.services-section.title')}</SectionTitle>
          {servicesData.map((service: ServiceContent, index: number) => (
            <CardContainer
              key={index}
            >
              <CardHeadline>{t(`services-ventures.services-section.section-title-${index}`, { text: service.headline })}</CardHeadline>
              <CardDescription>{t(`services-ventures.services-section.section-description-${index}`, { text: service.description })}</CardDescription>
            </CardContainer>
          ))}
        </SectionContainer>
        <SectionContainer>
          <SectionTitle>{t('services-ventures.ventures-section.title')}</SectionTitle>
          {venturesData.map((service: VentureContent, index: number) => (
            <CardContainer
              key={index}
            >
              <CardHeadline>{t(`services-ventures.ventures-section.section-title-${index}`, { text: service.headline })}</CardHeadline>
              <CardDescription>{t(`services-ventures.ventures-section.section-description-${index}`, { text: service.description })}</CardDescription>
            </CardContainer>
          ))}
        </SectionContainer>
      </ContentContainer>
    </Container>
  )
}

export default React.memo(ServicesAndVentures)
