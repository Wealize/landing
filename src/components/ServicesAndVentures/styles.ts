import styled from 'styled-components'

import { halfPx, singlePx, singleAndHalfPx, doublePx, doubleAndHalfPx, triplePx, quadruplePx } from '../../theme/space'
import { media } from '../../theme/media'
import { CONTAINER_SPACING } from '../../theme/spacingGuide'
import { WHITE_COLOR, GRAY_300_COLOR } from '../../theme/color'


export const Container = styled.div`
  display: flex;
  width: 100%;
  flex-direction: column;
  justify-content: center;
  background-color: ${WHITE_COLOR};
  ${CONTAINER_SPACING.MOBILE}

  ${media.greaterThan('md')`
    ${CONTAINER_SPACING.TABLET}
  `};

  ${media.greaterThan('lg')`
    ${CONTAINER_SPACING.SMALL_DESKTOP};
  `};

  ${media.greaterThan('xl')`
    ${CONTAINER_SPACING.DESKTOP};
  `};

  ${media.greaterThan('xxl')`
    ${CONTAINER_SPACING.WIDE};
  `};
`

export const Title = styled.h3`
  font-size: 1.8rem;
  line-height: 1.40;
  padding: 0;
  z-index: 1;

  ${media.greaterThan('md')`
    width: 70%;
  `};

  ${media.greaterThan('lg')`
    width: 60%;
  `};

  ${media.greaterThan('xl')`
    width: 55%;
  `};
`

export const ContentContainer = styled.div`
  padding: ${singlePx()} 0;
  display: flex;
  flex-direction: column;
  flex-wrap: wrap;

  ${media.greaterThan('md')`
    display: grid;
    grid-template-columns: repeat(2, 1fr);
    grid-gap: ${doublePx()};
  `};

  ${media.greaterThan('lg')`
    grid-gap: ${doubleAndHalfPx()};
  `};

  ${media.greaterThan('xl')`
    grid-gap: ${triplePx()};
  `};

  ${media.greaterThan('xxl')`
    grid-gap: ${quadruplePx()};
  `};
`

export const SectionContainer = styled.div`
  z-index: 1;
`

export const SectionTitle = styled.h4`
  text-transform: uppercase;
  font-size: 1.1rem;
  font-weight: bold;
  padding-top: ${doubleAndHalfPx()};
  letter-spacing: 1.44px;
  z-index: 1;
`

export const CardContainer = styled.div`
  padding-top: ${singleAndHalfPx()};
`

export const CardHeadline = styled.div`
  font-size: 1.6rem;
  line-height: 1.52;
  padding: ${halfPx()} 0;
  z-index: 1;

  ${media.greaterThan('lg')`
    font-size: 1.8rem;
  `};
`

export const CardDescription = styled.h5`
  font-size: 1.1rem;
  line-height: 1.52;
  color: ${GRAY_300_COLOR};
  z-index: 1;

  ${media.greaterThan('xl')`
    font-size: 1.1rem;
  `};
`
