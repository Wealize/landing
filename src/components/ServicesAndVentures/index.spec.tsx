import React from 'react'
import { mount } from 'enzyme'


import { Title, SectionTitle, SectionContainer, CardContainer } from './styles'
import ServicesAndVentures from './index'

describe('<ServicesAndVentures />', () => {
  const wrapper = mount(<ServicesAndVentures />)

  it('should render', () => {
    expect(wrapper).toBeDefined()
    expect(wrapper).toMatchSnapshot()
  })

  it('should render content correctly', () => {
    expect(wrapper.find(Title).text().length).toBeGreaterThan(0)
    expect(wrapper.find(SectionTitle).first().text().length).toBeGreaterThan(0)
    expect(wrapper.find(SectionContainer).length).toEqual(2)
    expect(wrapper.find(CardContainer).length).toBeGreaterThan(0)
  })
})
