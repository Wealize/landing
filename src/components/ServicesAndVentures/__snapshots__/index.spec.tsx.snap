// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ServicesAndVentures /> should render 1`] = `
<Memo(ServicesAndVentures)>
  <styled.div>
    <div
      className="mPdND"
    >
      <styled.h3>
        <h3
          className="gxcBLJ"
        >
          profile:services-ventures.title
        </h3>
      </styled.h3>
      <styled.div>
        <div
          className="cySEK"
        >
          <styled.div>
            <div
              className=""
            >
              <styled.h4>
                <h4
                  className="eDfkbp"
                >
                  profile:services-ventures.services-section.title
                </h4>
              </styled.h4>
              <styled.div
                key="0"
              >
                <div
                  className="cGNQSf"
                >
                  <styled.div>
                    <div
                      className="kIpVJm"
                    >
                      profile:services-ventures.services-section.section-title-0
                    </div>
                  </styled.div>
                  <styled.h5>
                    <h5
                      className="JvsHx"
                    >
                      profile:services-ventures.services-section.section-description-0
                    </h5>
                  </styled.h5>
                </div>
              </styled.div>
              <styled.div
                key="1"
              >
                <div
                  className="cGNQSf"
                >
                  <styled.div>
                    <div
                      className="kIpVJm"
                    >
                      profile:services-ventures.services-section.section-title-1
                    </div>
                  </styled.div>
                  <styled.h5>
                    <h5
                      className="JvsHx"
                    >
                      profile:services-ventures.services-section.section-description-1
                    </h5>
                  </styled.h5>
                </div>
              </styled.div>
              <styled.div
                key="2"
              >
                <div
                  className="cGNQSf"
                >
                  <styled.div>
                    <div
                      className="kIpVJm"
                    >
                      profile:services-ventures.services-section.section-title-2
                    </div>
                  </styled.div>
                  <styled.h5>
                    <h5
                      className="JvsHx"
                    >
                      profile:services-ventures.services-section.section-description-2
                    </h5>
                  </styled.h5>
                </div>
              </styled.div>
            </div>
          </styled.div>
          <styled.div>
            <div
              className=""
            >
              <styled.h4>
                <h4
                  className="eDfkbp"
                >
                  profile:services-ventures.ventures-section.title
                </h4>
              </styled.h4>
              <styled.div
                key="0"
              >
                <div
                  className="cGNQSf"
                >
                  <styled.div>
                    <div
                      className="kIpVJm"
                    >
                      profile:services-ventures.ventures-section.section-title-0
                    </div>
                  </styled.div>
                  <styled.h5>
                    <h5
                      className="JvsHx"
                    >
                      profile:services-ventures.ventures-section.section-description-0
                    </h5>
                  </styled.h5>
                </div>
              </styled.div>
              <styled.div
                key="1"
              >
                <div
                  className="cGNQSf"
                >
                  <styled.div>
                    <div
                      className="kIpVJm"
                    >
                      profile:services-ventures.ventures-section.section-title-1
                    </div>
                  </styled.div>
                  <styled.h5>
                    <h5
                      className="JvsHx"
                    >
                      profile:services-ventures.ventures-section.section-description-1
                    </h5>
                  </styled.h5>
                </div>
              </styled.div>
              <styled.div
                key="2"
              >
                <div
                  className="cGNQSf"
                >
                  <styled.div>
                    <div
                      className="kIpVJm"
                    >
                      profile:services-ventures.ventures-section.section-title-2
                    </div>
                  </styled.div>
                  <styled.h5>
                    <h5
                      className="JvsHx"
                    >
                      profile:services-ventures.ventures-section.section-description-2
                    </h5>
                  </styled.h5>
                </div>
              </styled.div>
            </div>
          </styled.div>
        </div>
      </styled.div>
    </div>
  </styled.div>
</Memo(ServicesAndVentures)>
`;
